sinonjs/samsam
# Move to CircleCI

This PR moves our CI from Travis to CircleCI

#### Purpose

Make tests run in node 10.

For whatever reason, the node 10 in Travis doesn't support `require.resolve` as can be seen in the failed builds: https://travis-ci.org/sinonjs/samsam/builds/579155672

The same node version works fine locally (macOS).


#### Solution  - optional

Move to CircleCI, which I've wanted to do for awhile anyway (because of Travis' unethical treatment of workers, some of whom are my friends).

#### How to verify - mandatory

1. Verify that tests pass in CircleCI
1. Observe that the Travis configuration has been removed

#### Checklist for author
